ui/src/extractedTranslations/fr/translation.json

```diff
@@ -64,19 +64,19 @@
     "face_group": {
       "label_placeholder": "Étiquette",
       "unlabeled": "Sans étiquette",
-      "unlabeled_person": null
+      "unlabeled_person": "Personne sans étiquette"
     },
     "modal": {
       "action": {
-        "merge": null
+        "merge": "Fusionner"
       },
       "detach_image_faces": {
         "action": {
-          "detach": null,
-          "select_images": null
+          "detach": "Détacher les images de visage",
+          "select_images": "Sélectionner les images de visage à détacher"
         },
         "description": null,
-        "title": null
+        "title": null,
```

Comment:
The comma at the end is causing a syntax error

Reply:
Oh sorry, I'm relative new with Github